solr/core/src/java/org/apache/solr/handler/IndexFetcher.java

```diff
@@ -345,8 +346,13 @@ public NamedList<Object> getLatestVersion() throws IOException {
     ModifiableSolrParams params = new ModifiableSolrParams();
     params.set(COMMAND, CMD_INDEX_VERSION);
     params.set(CommonParams.WT, JAVABIN);
-    params.set(CommonParams.QT, ReplicationHandler.PATH);
-    QueryRequest req = new QueryRequest(params);
+    var req =
```

Comment:
[Q] SolrJ has a SolrRequest for the v2 equivalent of this API: ReplicationApi.FetchIndexVersion.  If this code is targeting 10.0 only, wdyt of using that here instead of GSR?

Reply:
Tried switching to FetchIndexVersion and the change looks great, but ultimately decided against it until we agree on a path forward for [SOLR-17549](https://issues.apache.org/jira/browse/SOLR-17549).